modules/multiplayer/doc_classes/SceneReplicationConfig.xml

```diff
@@ -37,49 +37,64 @@
 				Finds the index of the given [param path].
 			</description>
 		</method>
+		<method name="property_get_replication_mode">
+			<return type="int" enum="SceneReplicationConfig.ReplicationMode" />
+			<param index="0" name="path" type="NodePath" />
+			<description>
+				Returns the replication mode for the property identified by the given [param path]. See [enum ReplicationMode].
+			</description>
+		</method>
 		<method name="property_get_spawn">
 			<return type="bool" />
 			<param index="0" name="path" type="NodePath" />
 			<description>
 				Returns whether the property identified by the given [param path] is configured to be synchronized on spawn.
 			</description>
 		</method>
-		<method name="property_get_sync">
+		<method name="property_get_sync" is_deprecated="true">
 			<return type="bool" />
 			<param index="0" name="path" type="NodePath" />
 			<description>
 				Returns whether the property identified by the given [param path] is configured to be synchronized on process.
 			</description>
 		</method>
-		<method name="property_get_watch">
+		<method name="property_get_watch" is_deprecated="true">
 			<return type="bool" />
 			<param index="0" name="path" type="NodePath" />
 			<description>
 				Returns whether the property identified by the given [param path] is configured to be reliably synchronized when changes are detected on process.
 			</description>
 		</method>
-		<method name="property_set_spawn">
+		<method name="property_set_replication_mode">
+			<return type="void" />
+			<param index="0" name="path" type="NodePath" />
+			<param index="1" name="mode" type="int" enum="SceneReplicationConfig.ReplicationMode" />
+			<description>
+				Sets the synchronization mode for the property identified by the given [param path]. See [enum ReplicationMode].
+			</description>
+		</method>
+		<method name="property_set_spawn" is_deprecated="true">
```

Comment:
Why set spawn is deprecated, but get spawn is not?

Reply:
Ah, nice spot, that was a mistake indeed, should have been added to `property_get/set_watch` and `property_get/set_sync` only.

Fixed.